src/libraries/System.Net.Sockets/tests/FunctionalTests/SelectTest.cs

```diff
@@ -417,7 +417,12 @@ private static void DoAccept(Socket listenSocket, int connectionsToAccept)
                 }
             }
         }
+    }
 
+    [Collection(nameof(DisableParallelization))]
```

Comment:
If this is exhausting process-wise file descriptors, instead of disabling parallelization on this test, should we run this inside `RemoteExecutor`?

Reply:
I'm not sure if the limits are per process or per user. We may try both if this fails or we can disable it explicitly on `tvOS` and `Catalyst`